### PR TITLE
Endpoint train model encoding fix

### PR DESCRIPTION
**Pull request by mihail911**
fixed encoding error in endpoint as per @akhilaananthram's suggestion. Please review @BoltzmannBrain

### CODE REVIEW

File: fluent/models/classify_endpoint.py [hunk 1]
```diff
@@ -110,16 +110,17 @@ def trainModel(self, samples, labels, negatives=None):
     labels_to_update_bitmaps = set()
```
Comment (BoltzmannBrain) [inline fluent/models/classify_endpoint.py]: Please stick to our naming conventions... `labelsToUpdateBitmap`. Make sure the rest of the variables and methods in this file follow the convention.

File: fluent/models/classify_endpoint.py [hunk 2]
```diff
@@ -110,16 +110,17 @@ def trainModel(self, samples, labels, negatives=None):
...
     for sample,sample_labels in zip(samples, labels):
```
Comment (BoltzmannBrain) [inline fluent/models/classify_endpoint.py]: space after comma

**Comment by BoltzmannBrain** [comment]
@mihail911 left a couple nitpicks. Have the changes to this model been validated by testing on the sample dataset against the expected classes?

**Comment by mihail911** [comment]
@BoltzmannBrain validated against sample dataset and made your suggested changes

**Comment by BoltzmannBrain** [comment]
merged w/ #115

**Pull request merged**